src/test/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupportSpec.scala

```diff
@@ -23,8 +23,10 @@ class TSVFileSupportSpec extends AnyFreeSpec with TSVFileSupport {
       val attributes = getWorkspaceAttributeCalls(MockTSVLoadFiles.validWorkspaceAttributes)
       assertResult(attributes) {
         List(AddUpdateAttribute(AttributeName("default", "a1"), AttributeString("v1")),
-          AddUpdateAttribute(AttributeName("default", "a2"), AttributeString("2")),
-          AddUpdateAttribute(AttributeName("default", "a3"), AttributeString("[1,2,3]")))
+          AddUpdateAttribute(AttributeName("default", "a2"), AttributeValueRawJson("2")),
```

Comment:
Note these are changed from `AttributeString` to `AttributeValueRawJson`; see question in the PR description.